File: ama/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_latlong/Results.java
package com.android.blawniczak.astroweather.domain.woeid_latlong;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Results {

    @SerializedName("place")
    @Expose
    public Place place;

}

File: paiwjs/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_latlong/Results.kt
package com.android.blawniczak.astroweather.domain.woeid_latlong

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Results {

    @SerializedName("place")
    @Expose
    var place: Place? = null

}
